src/documentation/programming-guide.md

```diff
@@ -319,9 +415,18 @@ PCollection<Integer> wordLengths = words.apply(
     }));
 ```
 
-If your `ParDo` performs a one-to-one mapping of input elements to output elements--that is, for each input element, it applies a function that produces *exactly one* output element, you can use the higher-level `MapElements` transform. `MapElements` can accept an anonymous Java 8 lambda function for additional brevity.
+```python
+# The input PCollection of Strings.
+words = ...
+
+# Apply a ParDo with an lambda DoFn to the PCollection words.
+# Save the result as the PCollection wordLengths.
+wordLengths = words | beam.ParDo(lambda x: [len(x)])
```

Comment:
In Python `ParDo` only accepts a `DoFn`. `FlatMap` takes a `callable`

```
# The input PCollection of strings.
words = ...

# Apply a lambda function to the PCollection words.
# Save the result as the PCollection word_lengths.
word_lengths = words | beam.FlatMap(lambda x: [len(x)])
```

(Also note that string is not capitalized)